search-commons/src/main/java/no/unit/nva/search2/importcandidate/Constants.java

```diff
@@ -102,4 +105,32 @@ public static NestedAggregationBuilder topLevelOrganisationsHierarchy() {
     @JacocoGenerated
     public Constants() {
     }
+
+    public static Script selectByLicense(String license) {
+        var script = """
+            if (doc['associatedArtifacts.license.keyword'].size()==0) { return false;}
+            def url = doc['associatedArtifacts.license.keyword'].value;
+            if (url.contains("/by-nc-nd")) {
+              return "CC-NC-ND".equals(params.license);
+            } else if (url.contains("/by-nc-sa")) {
+              return "CC-NC-SA".equals(params.license);
+            } else if (url.contains("/by-nc")) {
+              return "CC-NC".equals(params.license);
+            } else if (url.contains("/by-nd")) {
+              return "CC-ND".equals(params.license);
+            } else if (url.contains("/by-sa")) {
+              return "CC-SA".equals(params.license);
+            } else if (url.contains("/by")) {
+              return "CC-BY".equals(params.license);
+            } else {
+                return "Other".equals(params.license);
+            }
+            """;
+        return new Script(
+            ScriptType.INLINE,
+            no.unit.nva.search2.resource.Constants.PAINLESS,
+            script,
+            Map.of("license", license.toUpperCase(Locale.getDefault()))
+        );
+    }
```

Comment:
Do we still need this?

Reply:
No, it is going away, but not in this qa.